WordPress/gutenberg
# Keyboard Shortcuts: fix settings sidebar toggle shortcut

Part of #43352

## What?
This PR fixes settings sidebar toggle shortcut.

## Why?
`Ctrl + Shift + ,` (on Windows) should open and close the settings sidebar, as shown in the window below:

![keyboard-modal](https://user-images.githubusercontent.com/54422211/185630449-7725178d-ed40-4a5e-af56-ca5f6ce3267a.png)

However, as described in [the MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key), KeyboardEvent.key returns a value that takes into consideration the state of modifier keys.

> The [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) interface's key read-only property returns the value of the key pressed by the user, taking into consideration the state of modifier keys such as Shift as well as the keyboard locale and layout.

Therefore, in Windows, `,` key is detected as the `<` key:

https://user-images.githubusercontent.com/54422211/185631089-d75be8b5-ccbb-44fa-8a7c-a98d7164f248.mp4


## How?
The keyboard shortcut modal should display key labels that don't take modifier keys into consideration.
Therefore, I replaced `<` with `,` in the detection condition by `isKeyboardEvent`.
As far as I know, `<` and `,` are not used as other keyboard events, so I think replacing them should have no effect.
In addition, this fix does not make a judgment based on the operating system. If this was not a problem on the Mac, I would like to exclude Macby `isAppleOS()`.

## Testing Instructions
- Open the post /site editor.
- Select a block, and press `Ctrl + Shift + ,` (Or shortcuts on the Mac)
- Confirm that the settings sidebar opens and closes.

https://user-images.githubusercontent.com/54422211/185634317-48c185bc-cb34-49c0-89b5-dca50d7481b9.mp4
